provisioning/e2e/_service_create_delete.js

```diff
@@ -92,4 +94,29 @@ testDescription.forEach(function(testConfiguration) {
       });
     });
   });
-});
\ No newline at end of file
+});
+
+testSpecification.forEach(function(testConfiguration) {
+  describe('#Update', function() {
+    this.timeout(10000);
+    var enrollmentToDelete = {};
+    after(function(done) {
+      testConfiguration.deleteFunction(enrollmentToDelete, function(err) {
+        assert.isNull(err,'Non null response from the delete AFTER create.');
+        done();
+      });
+    });
+    it(testConfiguration.testDescription, function(callback) {
+      enrollmentToDelete = {};
+      testConfiguration.createFunction(testConfiguration.enrollmentObject, function(err, returnedEnrollment) {
```

Comment:
>createFunction [](start = 24, length = 14)

shouldn't creation be in a before() clause?

Reply:
It's a bit of a pain to propagate the value through to the test.

---
In reply to: [154165590](https://github.com/Azure/azure-iot-sdk-node/pull/165#discussion_r154165590) [](ancestors = 154165590)

Reply:
Fixed it.

---
In reply to: [154196642](https://github.com/Azure/azure-iot-sdk-node/pull/165#discussion_r154196642) [](ancestors = 154196642,154165590)